less/services.less

```diff
@@ -0,0 +1,92 @@
+.serv-header {
+    height: 250px;
+    margin-top: 50px;
+    background-image: url('/img/services/services-jumbotron.png');
+    background-size: cover;
+    display: flex;
+    width: 100%;
+
+    .h1-div{
+        display: flex;
+        width: 80%;
+        justify-content: flex-start;
+        align-items: flex-end;
+        margin: 0 auto;
+        
+        h1 {
+            margin-bottom: 10px;
+
+        }
+    }
+
+}
+
+.container {
+    p {
+        margin-top: 50px;
+        margin-bottom: 50px;
+    }
+
+}
+
+.service-tabs {
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        
+
+    .tab {
+        font-size: 2.4rem;
+        font-weight: bold;
+        padding: 15px;
+        border: black solid 1px;
+        cursor: pointer;
+
+        &.tab-selected {
+            background-color: #5E9FB9;
+            color: white;
+        }
+
+        &:hover{
+            background-color: #5E9FB9;
+            color: white; 
+        }
+    }
+}
+
+.tab-content{
+    width: 100%;
+    display: flex;
+}
+
+.content  {
+    display: none;
+
+    &.shown{
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+        margin: 50px auto;
+
+        .item-cont{
+            display: flex;
+            width: 100%;
+
+            .para{
+                width: 50%;
+
+                p {
```

Comment:
this is probably more nesting than you really need on a site, Josh's rule of thumb is not to go over 4